az-digital/az_quickstart
# Closes #2169 Close "x" icon broken in media library edit module's modal/dialog

## Description
See #2169
Adding patch from https://www.drupal.org/project/drupal/issues/3335935

## Related issues
Closes #2169

## How to test

1. Create a new flexible page
2. Add a card deck
3. Add another paragraph type with media, such as Text on Media
4. Add an image to the second paragraph
5. Click the pencil "edit" icon to view the incorrect appearance for the close icon
6. Reload the page
7. Edit the second paragraph, edit the media item, and observe the correct close icon
8. Click to edit the card deck, then return to edit the media item in the second paragraph and observe the correct icon

## Types of changes

### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
